component/src/main/java/io/siddhi/extension/io/cdc/source/CDCSource.java

```diff
@@ -402,7 +404,11 @@ protected ServiceDeploymentInfo exposeServiceDeploymentInfo() {
                 validateListeningModeParameters(optionHolder);
 
                 //send sourceEventListener and preferred operation to changeDataCapture object
-                changeDataCapture = new ChangeDataCapture(operation, sourceEventListener);
+                if (url.contains("jdbc:mongodb")) {
```

Comment:
shall we check this in case insensitive way

Reply:
Fixed.